src/services/dashboards/dashboards.svc.coffee

```diff
@@ -30,6 +30,10 @@ angular
     @dashboardChanged = ->
       return _self.callbacks.dashboardChanged.promise
 
+    @callbacks.dashboardUpdated = $q.defer()
```

Comment:
If the currency change notification is not used anymore, I suggest removing the callback to avoid having unused code in the service